tables/fi.utb

```diff
@@ -2,28 +2,33 @@
 #
 # -----------
 #-index-name: Finnish
-#-display-name: Finnish braille
+#-display-name: Suomenkielinen 6-pisteen pistekirjoitustaulukko
 #
-#+language:fi
-#+type:literary
-#+contraction:no
-#+dots:6
-# Marked as "direction:forward" by Bue Vester-Andersen
-# as tests only run forward.
-#+direction:forward
+#+language: fi
+#+type: literary
+#+contraction: no
+#+dots: 6
+#+direction: forward
+#+authority: Finnish delegation for braille <braille@celia.fi>
 #
-# TODO: Please correct the metadata above. It is not meant to be
-# accurate nor complete. It hasn't been verified by the table
-# author yet. It is merely an attempt by the liblouis maintainers
-# to get some sensible initial values in place.
+#
+#-maintainer: Sami Määttä <sami.maatta@celia.fi>
+#-maintainer: Finnish delegation for braille <braille@celia.fi>
+#-author: Sami Määttä <sami.maatta@celia.fi>
+#-author: Kirsi Ylänne <kirsi.ylanne@celia.fi>
+#-author: Jukka Eerikäinen (created by)
+#-copyright: 2014-2024, Accessibility Library Celia, https://www.celia.fi/en
```

Comment:
It's more correct to format the copyright years as "2014, 2015, 2024".

Reply:
Oh, and put the https://www.celia.fi/en inside brackets.

Reply:
Resolved with newest commits.